### PR TITLE
pip, pip-install, pip-uninstall, touch: add pt_BR translation

### PR DESCRIPTION
**Pull request by luis-saes**
- [X] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [X] The page(s) have at most 8 examples.
- [X] The page description(s) have links to documentation or a homepage.
- [X] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).

### CODE REVIEW

**Review by kbdharun — CHANGES_REQUESTED** [review]
I just have a suggestion that needs to be addressed.

File: pages.pt_BR/common/touch.md [hunk 1]
```diff
@@ -1,21 +1,24 @@
...
 
-- Criar um novo arquivo vazio, ou atualizar as timestamps para a hora atual:
+- Criar novo(s) arquivo(s) vazio(s) ou alterar os timestamps do(s) arquivo(s) para o timestamp atual:
 
 `touch {{arquivo}}`
 
-- Definir as timestamps de um arquivo para a hora especificada:
+- Definir os timestamps de um arquivo para uma data e hora específica:
 
 `touch -t {{YYYYMMDDHHMM.SS}} {{arquivo}}`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/common/touch.md:12]:
```suggestion
`touch -t {{YYYYMMDDHHMM.SS}} {{caminho/para/arquivo}}`
```

File: pages.pt_BR/common/touch.md [hunk 2]
```diff
@@ -1,21 +1,24 @@
...
 
-> Atualizar as timestamps de um arquivo para a hora atual.
-> Se o arquivo não existir, cria um arquivo vazio, a menos que seja passado o parâmetro -c ou -h.
+> Alterar os timestamps de acesso e de modificação (atime, mtime) de um arquivo.
 > Mais informações: <https://www.gnu.org/software/coreutils/touch>.
 
-- Criar um novo arquivo vazio, ou atualizar as timestamps para a hora atual:
+- Criar novo(s) arquivo(s) vazio(s) ou alterar os timestamps do(s) arquivo(s) para o timestamp atual:
 
 `touch {{arquivo}}`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/common/touch.md:8]:
```suggestion
`touch {{caminho/para/arquivo}}`
```

File: pages.pt_BR/common/touch.md [hunk 3]
```diff
@@ -1,21 +1,24 @@
...
 
-- Definir as timestamps de um arquivo para a hora especificada:
+- Definir os timestamps de um arquivo para uma data e hora específica:
 
 `touch -t {{YYYYMMDDHHMM.SS}} {{arquivo}}`
 
-- Usar as timestamps do arquivo1 para definir as timestamps do arquivo2:
+- Definir os timestamps de um arquivo para uma hora no passado:
+
+`touch -d "{{-1 hour}}" {{arquivo}}`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/common/touch.md:16]:
```suggestion
`touch -d "{{-1 hour}}" {{caminho/para/arquivo}}`
```

File: pages.pt_BR/common/touch.md [hunk 4]
```diff
@@ -1,21 +1,24 @@
...
+
+- Usar as timestamps de um arquivo para definir as timestamps de um segundo arquivo:
 
 `touch -r {{arquivo1}} {{arquivo2}}`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/common/touch.md:20]:
```suggestion
`touch -r {{caminho/para/arquivo1}} {{caminho/para/arquivo2}}`
```

File: pages.pt_BR/common/touch.md [hunk 5]
```diff
@@ -1,21 +1,24 @@
...
 
-- Alterar as timestamps de um arquivo. Não cria novo arquivo se não existir:
+- Criar múltiplos arquivos:
 
-`touch -c {{arquivo}}`
+`touch -c {{arquivo{1,2,3}.txt}}`
```
Comment (EmilySeville7cfg) [inline pages.pt_BR/common/touch.md:24]:
```suggestion
`touch -c {{arquivo{1,2,3}.txt}}`
```

It is not written in our style guide but we don't prefer brace expansion-like syntax in placeholders. The original page is going to be refreshed by [this](https://github.com/tldr-pages/tldr/pull/7966) PR. But now it's the correct translation.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]

**Review by EmilySeville7cfg — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
Merged, thanks! ❤️

**Comment by luis-saes** [comment]
Thank you, guys 😁

**Pull request merged**